articles/hdinsight/domain-joined/general-guidelines.md

```diff
@@ -69,6 +69,28 @@ If the url auth is enabled:
 * If the access is for this url, then Ranger will check if the user is in the allow list.
 * Ranger won't check any of the fine grained policies.
 
+### Managing Ranger Audit Logs
+
+To prevent Ranger audit logs from consuming too much disk space on your hn0 headnode, you can change the number of days to retain the logs. 
+
+1. Sign in to the **Ambari UI**.
+2. Navigate to **Services** -> **Ranger** -> **Configs** -> **Advanced** -> **Advanced ranger-solr-configuration**.
+3. Change the 'Max Retention Days' to 7 days or less.
+4. Click **Save** and restart affected components for the change to take effect. 
+
+### Use a custom Ranger DB  
+
+We recommend deploying an external Ranger DB to use with your ESP cluster for high-availability of Ranger metadata, which ensures that policies are available even if the cluster is unavailable. Since an external database is customer-managed, you will also have the ability to tune the DB size and share the database across multiple ESP clusters. You can specify your [external Ranger DB during the ESP cluster creation](https://learn.microsoft.com/azure/hdinsight/hdinsight-hadoop-provision-linux-clusters) process using the Azure Portal, ARM, Azure CLI, etc. 
```

Comment:
```suggestion
We recommend deploying an external Ranger DB to use with your ESP cluster for high availability of Ranger metadata, which ensures that policies are available even if the cluster is unavailable. Since an external database is customer-managed, you'll also have the ability to tune the DB size and share the database across multiple ESP clusters. You can specify your [external Ranger DB during the ESP cluster creation](/azure/hdinsight/hdinsight-hadoop-provision-linux-clusters) process using the Azure portal, Azure Resource Manager, Azure CLI, etc. 
```